frontend/pages/index.vue

```diff
@@ -1,24 +1,233 @@
-<script setup>
-// Demo für Routing zu lokalen Seiten und Sprachwechsel
-const { locale, setLocale } = useI18n();
-const localeRoute = useLocaleRoute();
+<script lang="ts" setup>
+import { useWindowScroll } from "@vueuse/core";
 
-// demo for data fetching
-const client = useSanctumClient();
+const { y } = useWindowScroll();
+
+const scrollToTopOfPage = () => {
+  y.value = 0;
+};
+
+const scrollToTopButtonIsVisible = computed(() => y.value > 200);
 </script>
 
 <template>
-  <div>
-    <div>
-      <button @click="setLocale('de')">de</button>
-      <br />
-      <button @click="setLocale('en')">en</button>
-      <p class="font-heading font-black">{{ $t("welcome") }}</p>
-      <NuxtLink
-        :to="localeRoute('play', locale)"
-        class="font-heading font-thin"
-        >{{ $t("join") }}</NuxtLink
+  <ClientOnly>
+    <div class="absolute top-12 z-10 flex w-screen justify-center lg:hidden">
```

Comment:
```suggestion
    <div class="absolute top-12 z-10 flex w-full justify-center lg:hidden">
```
es gibt noch immer einen overflow
![image](https://github.com/user-attachments/assets/0bb59c74-b2c3-4a55-915c-04079fed556d)